Makefile

```diff
@@ -7,6 +9,10 @@ build-image:
 	# save some time setting up the docker build context by deleting this first.
 	rm -f cluster-samples-operator        
 	docker build -t docker.io/openshift/origin-cluster-samples-operator:latest .
+build-image-dev:
+	rm -f cluster-samples-operator
+	docker build -t docker.io/$(REPO)/origin-cluster-samples-operator:latest .
+	docker push docker.io/$(REPO)/origin-cluster-samples-operator:latest
```

Comment:
i don't think this belongs here.

"make build-image && docker push foo" covers it, and this is going to fail by default for most people since they won't set REPO and can't push to the openshift repo.

also doing image pushing in a target named build is odd.

eventually we're probably going to want patterns for helping w/ these iterative workflows, but i don't think this is the one we want.

Updating the build-image target to use the new $(REPO) variable would be fine though.